docs/conf.py

```diff
@@ -28,6 +28,10 @@
 cwd = os.getcwd()
 project_root = os.path.dirname(cwd)
 
+# On Travis python3-gi is installed into folllowing directory, which
+# is not added to path because Travis uses virtualenv.
+sys.path.insert(0, '/usr/lib/python3/dist-packages')
```

Comment:
Probably because tox detects the global installation incorrectly.

There are multiple python installations:
* system python (`/usr/bin/python3.4`)
* travis’s (`/home/travis/virtualenv/python2.7_with_system_site_packages/bin/python`)
    `sys.path == ['', '/home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7', '/home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7/plat-x86_64-linux-gnu', '/home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7/lib-tk', '/home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7/lib-old', '/home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7/lib-dynload', '/usr/lib/python2.7', '/usr/lib/python2.7/plat-x86_64-linux-gnu', '/usr/lib/python2.7/lib-tk', '/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages', '/home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7/site-packages', '/usr/local/lib/python2.7/site-packages', '/usr/local/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages']`
* tox’s (`/home/travis/build/projecthamster/hamster-gtk/.tox/docs/bin/python`)
`sys.path == ['', '/home/travis/build/projecthamster/hamster-gtk', '/home/travis/build/projecthamster/hamster-gtk/hamster_gtk', '/home/travis/build/projecthamster/hamster-gtk/.tox/docs/lib/python35.zip', '/home/travis/build/projecthamster/hamster-gtk/.tox/docs/lib/python3.5', '/home/travis/build/projecthamster/hamster-gtk/.tox/docs/lib/python3.5/plat-linux', '/home/travis/build/projecthamster/hamster-gtk/.tox/docs/lib/python3.5/lib-dynload', '/opt/python/3.5.0/lib/python3.5', '/opt/python/3.5.0/lib/python3.5/plat-linux', '/home/travis/build/projecthamster/hamster-gtk/.tox/docs/lib/python3.5/site-packages', '/opt/python/3.5.0/lib/python3.5/site-packages']`
* ?'s (`/opt/python/3.5.0/bin/python`)
tox seems to recognize `/opt/python/3.5.0/bin/python` as global instalation.